### DISCUSSION TITLE
Welcome to Discussions!

### DISCUSSION THREAD

**Discussion by jamesturk** [openstates/issues#183] [General] [answered]
## 👋 Welcome!
  We’re moving from the old Discourse to GitHub Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LukeClancy** [comment, marked as answer]
Hello! My name is Luke Clancy, I was on the https://discourse.openstates.org, but that is now directing me here (if you make it http) so I guess there has been a migration? Or is it just down? What is happening with that?

I am working on some data analysis over the open states dataset, and am also working on a discussion website powered by it utilizing ROR. I will let open states know when either of these becomes useful! A recent graduate (not so recent anymore - last December) I live in Bremington WA and could use some contracting work

**Reply by jamesturk** [reply]
welcome Luke!  Yes, this is replacing the old Discourse

Glad to hear about your usage & meet you, welcome!
